.ci/travis.sh

```diff
@@ -69,6 +71,69 @@ checkstyle-tester-diff-groovy-patch-only)
     -pc my_check.xml -p patch-branch -r ../.ci-temp/checkstyle -m single
   ;;
 
+checkstyle-tester-diff-groovy-regression-single)
```

Comment:
The way i understand this, you run regression using Contribution Master and Contribution PR and check if the 2 separate runs produce the same or different files. I assume difference will fail the CI (and are we sure of that?).

If the above is correct, is this planned to be merged in? What happens if we make diff-report-util change the layout. This will then always fail CI even though it is an expected change.

Reply:
> I assume difference will fail the CI (and are we sure of that?).

Yes. https://travis-ci.org/github/checkstyle/contribution/jobs/763804426

> If the above is correct, is this planned to be merged in? What happens if we make diff-report-util change the layout. This will then always fail CI even though it is an expected change.

I would guess at that point we would need to use some sort of more complicated awk or perl command to verify that violations match up; right now, this CI job confirms that both the violations and format of the report are consistent.

Reply:
I wouldn't do anything too complicated. Should we not merge it or is there some alternative for this to not cause issues easily?

Reply:
I think we should keep these, I have pushed several times in this PR, and these jobs haven't been flaky. It is the best way that I can think of to easily check that the violations, reports, and report formats are all consistent.

I think it is ok to wait and see what the future brings regarding changes to `patch-diff-util` before doing anything different here.